packages/console/src/hooks/use-api.ts

```diff
@@ -25,22 +25,28 @@ const toastError = async (response: Response) => {
   }
 };
 
-const useApi = () => {
+type Parameters = {
+  hideErrorToast?: boolean;
+};
+
+const useApi = ({ hideErrorToast }: Parameters = {}) => {
   const { isAuthenticated, getAccessToken } = useLogto();
 
   const api = useMemo(
     () =>
       ky.create({
         hooks: {
-          beforeError: [
-            (error) => {
-              const { response } = error;
+          beforeError: hideErrorToast
+            ? []
+            : [
+                (error) => {
+                  const { response } = error;
 
-              void toastError(response);
+                  void toastError(response);
 
-              return error;
-            },
-          ],
+                  return error;
+                },
+              ],
```

Comment:
Still need to return error even if `hideErrorToast` is specified. So we should wrap the if condition check around `toastError`.
```typescript
beforeError: [
  (error) => {
    if (!hideErrorToast) {
      const { response } = error;
      void toastError(response);
    }
    return error;
  }
]
```

Reply:
The default behavior(without hook) is `return error`, so I think is better to leave it as default.